src/Filters/SingleCheckbox.jsx

```diff
@@ -3,12 +3,12 @@ import React from 'react';
 export default class SingleCheckbox extends React.Component {
   render() {
     return (
-      <section className="single-checkbox">
```

Comment:
How do we feel about single checks for HTML / JSX? Seems like this is a case where you'd stick to double quotes for actual HTML properties but single quotes for JavaScript-related strings for readability but I'm OK either way as long as we're consistent.